### PR TITLE
[BW-1143] Update ansi-html

### PR DESCRIPTION
**Pull request by kpierre13**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3228,7 +3228,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"ansi-html@npm:0.0.7, ansi-html@npm:^0.0.7":
+"ansi-html@npm:0.0.7":
```
Comment (cahrens) [inline yarn.lock]: I see that the 0.0.7 version is still used by "webpack-dev-server@npm:3.11.1", which should be fine since it is the dev server.

**Review by cahrens — APPROVED** [review]
LGTM, but I did not run with it.

File: yarn.lock (hunk 1 above)
Comment (cahrens) [reply yarn.lock]: But this means that the dependabot alert will need to be dismissed, as it will still complain about this usage.

**Review by cjllanwarne — APPROVED** [review]
LGTM assuming you can get the tests to pass 🤞

File: yarn.lock [hunk 2]
```diff
@@ -15944,7 +15953,7 @@ resolve@^2.0.0-next.3:
   version: 3.11.1
   resolution: "webpack-dev-server@npm:3.11.1"
   dependencies:
-    ansi-html: 0.0.7
+    ansi-html: 0.0.8
```
Comment (cahrens) [inline yarn.lock:15956]: @kpierre13 how did you get it to make this update? When I deleted the whole "webpack-dev-server@npm:3.11.1": block, it did not update in ansi-html after I did `yarn install` (following the directions in https://github.com/DataBiosphere/terra-ui/wiki/Security-and-Maintenance). I don't think we can just edit dependency versions ourselves, as the libraries define what versions they define on (and theoretically there could be an incompatibility with 0.0.8).

**Review by cahrens — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 2 above)
Comment (cahrens) [reply yarn.lock:15956]: In fact, when I go back to what is on dev and either follow the documented process or run `yarn up -R ansi-html` as suggested by @aednichols, neither of these usages update. So I don't think the upgrade can be done without upgrading the libraries that depend on ansi-html.

See the wiki document for options in this case.

Comment (kpierre13) [reply yarn.lock:15956]: I also ran that command and didn't see any changes, so I assumed that I must have to change it manually. Are the circleci checks not in place for version changes? How would I go about testing to make sure that these version upgrades don't break anything?

Comment (zarsky-broad) [reply yarn.lock:15956]: Notably, both instances of this are only used for serving a local dev environment, so we could close this warning as not used. Upgrading would require going to the new major version of `react-scripts`, which is a nontrivial undertaking.

Comment (cahrens) [reply yarn.lock:15956]: Correct, and this is documented in the wiki page (**A lot of the Dependabot alerts can be safely dismissed. These are all valid reasons for dismissing the alerts...**).

Comment (kpierre13) [reply yarn.lock:15956]: I understand. So if everyone agrees, I'm going to close this PR with the resolution to simply dismiss this alert due to its non-usage in production.

**Pull request closed without merging**